Scheduler - Getting Started: Add ASP.NET Core Snippets (#9038)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/00 Getting Started with Scheduler/10 Configure Views.md

DevExtreme Scheduler ships with the following view types:

* [Day](/concepts/05%20UI%20Components/Scheduler/060%20Views/010%20View%20Types/010%20Day%20View.md '/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Day_View')
* [Week and Work week](/concepts/05%20UI%20Components/Scheduler/060%20Views/010%20View%20Types/020%20Week%20and%20WorkWeek%20Views.md '/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Week_and_WorkWeek_Views')
* [Month](/concepts/05%20UI%20Components/Scheduler/060%20Views/010%20View%20Types/030%20Month%20View.md '/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Month_View')
* [Timeline Day, Month, Week, and Work week](/concepts/05%20UI%20Components/Scheduler/060%20Views/010%20View%20Types/040%20Timeline%20Views.md '/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Timeline_Views')
* [Agenda](/concepts/05%20UI%20Components/Scheduler/060%20Views/010%20View%20Types/050%20Agenda%20View.md '/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Agenda_View')

---

##### jQuery

> Specify the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array to define and customize Scheduler views. Configure views as objects to specify view options. To add views with default options, specify [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) values as strings in the **views[]** array.

Reviewer: ```suggestion
Use the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array to define and customize Scheduler views. Add views as configuration objects to specify view options. To use default options, add [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) values as strings to the **views[]** array.
```
